docs/design/services/dynamodb/high-level-library/README.md

```diff
@@ -0,0 +1,38 @@
+# Project Tenets (unless you know better ones)
+
+1. Meeting customers in their problem space allows them to deliver value
+   quickly.
+2. Meeting customer expectations drives usability.
+3. Discoverability drives usage.
+
+# Introduction
+
+This project provides a Dynamo DB experience that is easier to use and
+understand than can be provided by the generated Dynamo DB client. It
+provides object-oriented features, like persisting and updating Java
+data objects, as well as expressive data retrieval operations.
+
+# Project Goals
+
+1. The high-level library should be the ideal way to integrate with
+   Dynamo DB.
+2. The nouns and verbs used by the high-level library should match those
+   in Dynamo DB.
+3. The high-level library should use and build on the idiomatic norms in
+   the rest of the 2.x Java SDK.
+4. The high-level library should perform well in cold JVMs.
+5. Using the low-level Dynamo DB client should be a conscious choice by
+   the customer, not an accidental one.
```

Comment:
What does accidental mean in this case?

Reply:
There is probably a better word for what I'm getting at. Customers that use the low-level client should know there's a high-level client and intentionally have chosen the low-level one. They shouldn't be able to just use the low-level client and not know any better.

Reply:
Okay yeah, I figured that's what you were trying to say. I agree, I think we can find a better way to phrase this. Don't know one off the top of my head though. I'll update when I think of one!